src/authMappingUtils.js

```diff
@@ -76,3 +77,17 @@ export const userHasMethodOnAnyProject = (method, userAuthMapping = {}) => {
   const hasMethod = actionArrays.some(x => x.some(actionHasMethod));
   return hasMethod;
 };
+
+export const shouldDisplaySubmissionUIComponents = (project, userAuthMapping) => {
+  if (
+    !config.useArboristUI
```

Comment:
i would prefer not to import `config` into here, but rather move this `!config.useArboristUI` condiction out to `index.jsx`

`index.jsx` is already been importing a bunch of configs through `localconf` and `useArboristUI` can be add to them